www/src/docs/utilities/index.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 name: Utilities
-order: 5
+order: 9
```

Comment:
I can't think of a more intelligent way to page order than this. Anyone have any suggestions? This is pretty simple / straightforward but seems like there might be a better way.

Reply:
Yeah, this is how I always end up doing it TBH